sw/host/provisioning/ft_lib/src/lib.rs

```diff
@@ -333,22 +397,26 @@ pub fn run_ft_personalize(
     // Compare host and device generated certificates' hash values. Note that
     // the dev calculated hash is represented as a number with different
     // endiannes.
-    if received_hash
+    if !received_hash
         .data
         .as_bytes()
         .iter()
         .rev()
         .zip(hasher.finalize().iter())
         .all(|(a, b)| a == b)
     {
-        Ok(())
-    } else {
         log::error!(
             "Hash received from device: {:x?}",
             received_hash.data.as_bytes()
         );
-        Err(anyhow!("Host vs Device certs hash mismatch"))
+        return Err(anyhow!("Host vs Device certs hash mismatch"));
     }
+
+    let certs: [&Vec<u8>; 3] = [&tpm_ek_cert_bytes, &tpm_cek_cert_bytes, &tpm_cik_cert_bytes];
+    validate_certs_chain(
+        "./sw/device/silicon_creator/manuf/keys/fake/fake_ca.pem",
```

Comment:
Hardcoding this path is not ideal. One way to avoid this is to add to the `rust_library`: 
```rust rustc_env = { "fake_ca_path": "$(rootpath //sw/device/silicon_creator/manuf/keys/fake:fake_ca.pem) }"
```
and here replace the path with `env!("openocd_cmsis_dap_adapter_cfg")`.

Reply:
can you please elaborate ont his comment, it seems to have been misformatted/partially lost?

Reply:
Ah yes, the way you have this won't work b/c this path will be broken in the bazel sandbox. Another option is to:
1. add another cmd line arg with the path to the CA cert [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/provisioning/ft/src/main.rs;drc=30d7e787c753caaa03fe68a4a70da1bbcbc1d96f;l=71)
2. set the CA cert path [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/manuf/provisioning_inputs.bzl;drc=1d40a002b5d5ceb333cb26361a0c7a15db6e92f4;l=31) with `$(rootpath //sw/device/silicon_creator/manuf/keys/fake:fake_ca.pem)`
3. then add the arg to `run_ft_personalize` function [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/provisioning/ft/src/main.rs;drc=30d7e787c753caaa03fe68a4a70da1bbcbc1d96f;l=198)
4. use it in the `run_ft_personalize` function

I would prefer cmd line arg over envar for portability to different systems.